src/components/Header.js

```diff
@@ -0,0 +1,23 @@
+import styles from './header.module.css';
+import NavBar from './NavBar'
+
+function Header() {
+
+    const navLinks = {
+        "/" : "Home",
+        "/projects" : "Projects",
+        "/contact": "Contact"
+    }
```

Comment:
This is part of 1 some comments. I think making this an array from the start would be a bit easier. You wouldn't have to map it on https://github.com/jedwardmook/react-portfolio/pull/4/files?diff=split&w=1#diff-c25cbd82eca26fc9714051e991f00bea997778601a72fc7d3b49952c46c09e38R5 and the anchors are the same as the label.

```suggestion
    const navLinks = [
        "Home",
        "Projects",
        "Contact"
    ]
```